vendor/bam/src/lib.rs

```diff
@@ -1,17 +1,12 @@
 #![warn(unused_extern_crates, missing_debug_implementations, rust_2018_idioms)]
 #![deny(unsafe_code)]
 
-#[macro_use]
-extern crate serde_derive;
-#[macro_use]
-extern crate log;
-#[macro_use]
-extern crate debug_stub_derive;
-
 mod api;
 pub mod client;
 pub mod config;
 pub mod connection;
+
+#[macro_use]
```

Comment:
❓ is that on purpose?

Reply:
Yes, you have to export macros up until the root of the crate to use the `bam::` syntax.